dutch/dubbelspel/hoofdstuk1.md

```diff
@@ -0,0 +1,15 @@
+Sammenvatting Dubbelspel door Frank Martinus Arion. 
+
+Deze korte samenvatting is van Hoofdstuk 1 van een bekende boek uit Curacao. 
+
+Vier  mannen  komen  iedere  zondagmiddag  samen  op  het  schaduwrijke  erf  van taxichauffeur  Boeboe  Fiel,  die  in Wakota (een buitenwijk van Willemstad) woont. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.